django/api/services/spreadsheet_uploader_prep.py

```diff
@@ -353,22 +353,27 @@ def email_validator(df, *columns, **kwargs):
             }
     return result
 
-
-
 def validate_field_values(df, *columns, **kwargs):
     allowed_values = kwargs.get("fields_and_values")
     invalid_values = []
     
     result = {}
+    delimiter = kwargs.get("delimiter")
     for column in df.columns:
         if column in allowed_values:
             indices = []
             series = df[column]
             for index, value in series.items():
-                if str(value).upper() not in (item.upper() for item in allowed_values[column]) and value != '' and value is not None and not pd.isna(value):
-                    indices.append(index + kwargs.get("indices_offset", 0))
-                    if str(value) not in invalid_values:
-                        invalid_values.append(str(value))
+                if delimiter is not None:
+                    items = [item.strip() for item in value.split(delimiter)]
+                
+                for item in items:
```

Comment:
I think `items` will be `None` if there is no delimiter, so you probably should add `items = [value.strip()]` before the `if delimiter is not None` line